tests/boot/boot_from_pxe.pm

```diff
@@ -108,6 +118,20 @@ sub run {
     send_key 'ret';
     save_screenshot;
 
+    if (get_var('ARCH') eq 'aarch64') {
```

Comment:
we have check_var for this

Reply:
Yes, you are right. And according to your following comments, I think I can make it independent of ARCH as well.